app/assets/javascripts/components/autocomplete.js

```diff
@@ -133,5 +135,60 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     $select.parentNode.removeChild($select)
   }
 
+  Autocomplete.prototype.initAutoCompleteSearchTopics = function () {
+    var millerColumns = document.querySelector('miller-columns')
+    var topics = millerColumns.taxonomy.flattenedTopics
+    var topicBreadcrumbs = millerColumns.taxonomy.allTopicNames
+
+    var topicSuggestions = []
+
+    topicBreadcrumbs.forEach((breadcrumbs, index) => {
+      topicSuggestions.push({
+        index: index,
+        topic: breadcrumbs[breadcrumbs.length - 1].trim(),
+        breadcrumbs: breadcrumbs
+      })
+    })
+
+    new window.accessibleAutocomplete({ // eslint-disable-line no-new, new-cap
+      id: 'topics-autocomplete',
+      name: 'topics-autocomplete',
```

Comment:
Can avoid these being hardcoded at all? or take them from an element initialising this?

Reply:
The thing with the autocomplete component (the erb bit in our app) is that we're not generating a select element unless we pass options to the component. We're in this case where we're not enhancing on top of a set of suggestions, that's why I hardcoded them (as they're required by the autocomplete).

Reply:
If we're only putting it there because it needs it maybe go for something random?

Reply:
We could add [a helper function to deal with this](https://github.com/alphagov/govuk-frontend/blob/ad596de756ef51619c6e5bc8f22f3b93479773dd/src/common.js#L18), I didn't thought we're expecting to have more than one search topics per page.

Reply:
Updated to grab the id and name from as it's now enhancing an input element